content/1.md

```diff
@@ -1,8 +1,103 @@
 ---
-title: 1 - Windows powertools & Firefox extensions
-description: Ensimmäinen episodi juhuu!
-date: 2021-01-01
+title: '1 – Windows powertools & Firefox extensions 🦾'
```

Comment:
Still not sure whether the title should be in Finnish like other episodes. 🤔 Having all Finnish episodes have Finnish titles might be most logical. If we ever record any English episodes, they should of course have English titles. Thoughts, @Steellow?

Reply:
Idk if there's good translation for 'powertools' so English is good? 🤔 Or maybe finglish (Windows powertools ja Firefox lisäosat), imo both are good so you can decide.

Reply:
I think powertool = <em lang="fi">tehotyökalu</em> which I kind of like. 😀 I would vote for <em lang="fi">"1 – Windows-tehotyökalut & Firefox-lisäosat 🦾"</em>.

I'm going to change that because you said I can decide. 😄 I also have to update the description.

Reply:
Done. What do you think about the description now? It has quite much repetition with the title so idk. 😅 Also idk what would be better.

Reply:
I think it's great, no need to overthink it 😌 Also repetitiveness is often good